.github/workflows/pages-preview.yml

```
@@ -0,0 +1,61 @@
+          preview-branch: gh-pages
+          umbrella-dir: pr-preview
+
+      - name: Comment/update canonical preview URL (idempotent)
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          issue-number: ${{ github.event.number }}
+          body: |
+            PR Preview (canonical)
+            https://${{ github.repository_owner }}.github.io/${{ github.event.repository.name }}/pr-preview/pr-${{ github.event.number }}/
+          edit-mode: replace
+          body-includes: "PR Preview (canonical)"
```

actionlint (1.7.8)

input "body-includes" is not defined in action "peter-evans/create-or-update-comment@v4". available inputs are "append-separator", "body", "body-file", "body-path", "comment-id", "edit-mode", "issue-number", "reactions", "reactions-edit-mode", "repository", "token"

(action)